biz.aQute.launcher/src/aQute/launcher/Launcher.java

```diff
@@ -514,11 +514,16 @@ void startBundles(List<Bundle> tobestarted) throws Exception {
 		// Add all bundles that we've tried to start but failed
 		all.addAll(wantsToBeStarted);
 
+		int startOptions = Bundle.START_ACTIVATION_POLICY;
+		if (parms.eagerActivation) {
+			startOptions = 0;
```

Comment:
I guess instead of setting to 0, it may be more future-proof to AND-off the bit.